gen/gen.go

```diff
@@ -179,6 +205,34 @@ func (g *Gen) formatSource(src []byte) []byte {
 	return code
 }
 
+// Read the swaggo overrides
+func parseOverrides(r io.Reader) (map[string]string, error) {
+	overrides := make(map[string]string)
+	scanner := bufio.NewScanner(r)
+	overridesReplace := regexp.MustCompile(`replace\s+(\S+)\s+(\S+)`)
+	overridesSkip := regexp.MustCompile(`skip\s+(\S+)`)
+	overridesComment := regexp.MustCompile(`^//.*`)
+	nonWhitespace := regexp.MustCompile(`\S`)
+
+	for scanner.Scan() {
+		line := scanner.Text()
+
+		if parts := overridesReplace.FindStringSubmatch(line); parts != nil {
+			overrides[parts[1]] = parts[2] // parts[0] is the full line
+		} else if parts := overridesSkip.FindStringSubmatch(line); parts != nil {
+			overrides[parts[1]] = "" // parts[0] is the full line
+		} else if nonWhitespace.MatchString(line) && !overridesComment.MatchString(line) {
+			return nil, fmt.Errorf("could not parse override: '%s'", line)
+		}
```

Comment:
else if statements make code harder to follow. try to use the "fail fast" approach.